pkg/apis/planetscale/v2/vitesskeyspace_types.go

```diff
@@ -291,12 +291,42 @@ type VitessKeyspaceStatus struct {
 	// If Idle is True, the keyspace is not deployed in any cells, so it should
 	// be safe to turn down the keyspace.
 	Idle corev1.ConditionStatus `json:"idle,omitempty"`
-	// ReshardingInProgress indicates if there is a VReplication workflow in progress.
-	ReshardingInProgress corev1.ConditionStatus `json:"reshardingInProgress,omitempty"`
-	// ActiveWorkflows is a list of the current VReplication workflows in progress.
-	ActiveWorkflows []string `json:"activeWorkflows,omitempty"`
+	// ReshardingStatus is used to provide information about an ongoing resharding operation.
+	ReshardingStatus ReshardingStatus `json:"reshardingStatus,omitempty"`
+	// Conditions is a map of all VitessKeyspace specific conditions we want to set and monitor.
+	// It's ok for multiple controllers to add conditions here, and those conditions will be preserved.
+	Conditions map[VitessKeyspaceConditionType]VitessKeyspaceCondition `json:"conditions,omitempty"`
```

Comment:
Let's add a `FIXME: Convert this to a list for kubectl wait compatibility before merging the feature branch to master.`

Reply:
Done.